src/us/kbase/auth2/lib/Authentication.java

```diff
@@ -135,7 +135,7 @@ public class Authentication {
 	//TODO ZLATER POLICYIDS maintain admin settable list of required policy IDs, check on login & force choice page if missing. fail login or create if missing. Return missing with choice data
 	//TODO TEST test with gzip compression in header
 	
-	//TODO NOW DOCS better API docs. Also document JSON structures for HTML endpoints.
+	//TODO DOCS better API docs. Also document JSON structures for HTML endpoints.
```

Comment:
Not sure why I had `NOW` here, I use that to signify something I need to do before release